src/windows_sys.rs

```diff
@@ -245,6 +245,65 @@ pub struct CONTEXT {
     pub SegSs: u32,
     pub ExtendedRegisters: [u8; 512],
 }
+
+cfg_if::cfg_if! {
+    if #[cfg(target_arch = "arm")] {
```

Comment:
This should not need `cfg_if`, though it might require moving the `#[cfg(target_arch = "arm")]` onto conflicting structs maybe?
```suggestion
```

Reply:
I just put the cfg guard in `windows_sys.rs` for the whole `include!` directive.